### ISSUE TITLE
[Bug] Invoke-DbatoolsFormatter on unix adds DOS EOL 

### ISSUE THREAD

**Issue by jayvdb** [sqlcollaborative/dbatools#5790]
#### Host used

- [ ] powershell.exe
- [ ] ISE
- [ ] VS Code
- [x] Other (please specify)

```
Name                           Value
----                           -----
PSVersion                      6.2.0
PSEdition                      Core
GitCommitId                    6.2.0
OS                             Linux 5.1.7-1-default #1 SMP Tue Jun 4 07:56:54 UTC 2019 (55f2451)
Platform                       Unix
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

## Expected Behavior

On Linux, a .ps1 file should usually be stored with a unix EOL.

Ideally the reformatter keeps the same EOL as the input file.

I am new at using unix pwsh, so there may be some meta settings which I could configure to prevent this.

If so, it seems in line with the dbatools approach that it would notice the problem and tell me to fix my settings.

## Actual Behavior

A file is unix EOL is changed by the reformatter to use dos EOL.

**Comment by jayvdb** [comment]
I've only just started using pwsh & dbatools, mostly for its nice solution for https://github.com/PowerShell/PSScriptAnalyzer/issues/775 , and I have seen there were bugfixes regarding unix/dos etc for this. https://github.com/sqlcollaborative/dbatools/issues/4373 and https://github.com/sqlcollaborative/dbatools/issues/4409

This issue described above was found with the pwsh listed above, and dbatools installed today using `Install-Module`.

OTOH, my CI is using https://chocolatey.org/packages/dbatools , which claims the latest version is 0.9.834 , which doesnt make sense.  Anyway, on AppVeyor CI I need to run `Invoke-DbatoolsFormatter $_; dos2unix $_ 2>&1 | Out-Null` for each file, and then `git diff --ignore-space-at-eol --exit-code` at the end to see if there were any formatting improvements.

**Comment by niphlod** [comment]
confirmed, as for now it assumes the developer use it on windows platform. I'll fix it once dbatools 1.0 is released
